webapp/packages/core-bootstrap/package.json

```diff
@@ -52,6 +52,7 @@
     "@cloudbeaver/core-version": "^0",
     "@cloudbeaver/core-version-update": "^0",
     "@cloudbeaver/core-view": "^0",
+    "@cloudbeaver/core-website": "^0",
```

Comment:
it's not a core it's a plugin

Reply:
now i see the problem, that we can't use it in our core-* so leave it as it is for now